.github/workflows/collector-builder.yml

```
@@ -76,6 +112,54 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
 
+      - name: Create ansible vars
+        run: |
+          {
+            echo "---"
+            echo "stackrox_io_username: ${{ secrets.QUAY_STACKROX_IO_RW_USERNAME }}"
+            echo "stackrox_io_password: ${{ secrets.QUAY_STACKROX_IO_RW_PASSWORD }}"
+            echo "rhacs_eng_username: ${{ secrets.QUAY_RHACS_ENG_RW_USERNAME }}"
+            echo "rhacs_eng_password: ${{ secrets.QUAY_RHACS_ENG_RW_PASSWORD }}"
+            echo "collector_git_ref: ${{ github.ref }}"
+            echo "collector_builder_tag: ${{ env.COLLECTOR_BUILDER_TAG }}"
+          } > ${{ github.workspace }}/ansible/secrets.yml
```

Insisting on the usage of `cat` here, since I can't see my previous comment on it here
```suggestion
          cat << EOF > ${{ github.workspace }}/ansible/secrets.yml
            ---
            stackrox_io_username: ${{ secrets.QUAY_STACKROX_IO_RW_USERNAME }}
            stackrox_io_password: ${{ secrets.QUAY_STACKROX_IO_RW_PASSWORD }}
            rhacs_eng_username: ${{ secrets.QUAY_RHACS_ENG_RW_USERNAME }}
            rhacs_eng_password: ${{ secrets.QUAY_RHACS_ENG_RW_PASSWORD }}
            collector_git_ref: ${{ github.ref }}
            collector_builder_tag: ${{ env.COLLECTOR_BUILDER_TAG }}
          EOF
```